indico/modules/events/papers/models/call_for_papers.py

```diff
@@ -120,3 +120,7 @@ def user_competences(self):
     @property
     def announcement(self):
         return MarkdownText(paper_reviewing_settings.get(self.event, 'announcement'))
+
+    @property
+    def paper_reviewing_settings(self):
```

Comment:
Well I think its better to put the whole settings there rather then each setting that we need separately, I also added it in reviewing area [#2801](https://github.com/indico/indico/pull/2801/commits/eceec90006c294bb92f94a7578c1675aca2aa9ce)  to display reviewing deadlines.  But yeah I can add that bind thing.